Dockerfile

```diff
@@ -40,14 +41,20 @@ RUN CGO_ENABLED=0 GOARCH=${TARGETARCH} go build -o /build/dist/wg-portal \
   -tags netgo \
   cmd/wg-portal/main.go
 
+######
+# Export binaries
+######
+FROM scratch AS binaries
+COPY --from=builder /build/dist/wg-portal /
+
 ######
 # Final image
 ######
 FROM alpine:3.19
 # Install OS-level dependencies
 RUN apk add --no-cache bash curl iptables nftables openresolv
 # Setup timezone
-ENV TZ=Europe/Vienna
+ENV TZ=UTC
 # Copy binaries
 COPY --from=builder /build/dist/wg-portal /app/wg-portal
```

Comment:
what is the improvement of using another build stage here?

and if there is some kind of improvement, shouldnt that be: --from=binaries ?

Reply:
ah nvm - just seen the change in docker-publish.yml